index.html

```diff
@@ -0,0 +1,625 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css" />
+    <title>Newsweek</title>
+
+</head>
+<main>
```

Comment:
The `<main>` tag should be inside the `<body>` tag and it should wrap the main content of the website only, for the headings, and the footer they should be wrapped in their appropriate tags too: `<header>` and `<footer>`.